### DISCUSSION TITLE
News Cache [2023-10-23]

### DISCUSSION THREAD

**Discussion by amyegan** [vercel/community#4581] [Announcements]
The [Changelog](https://vercel.com/changelog) has been quiet while we all prepare for [Next.js Conf](https://nextjs.org/conf) 👀

You can find past updates and resources in the [Announcements](https://github.com/orgs/vercel/discussions/categories/announcements?discussions_q=category:Announcements) category.

**Comment by FindMalek** [comment]
Can't wait!
